### ISSUE TITLE
Unable to upgrade / scale sentry web pod

### ISSUE THREAD

**Issue by Hassnain-Alvi** [sentry-kubernetes/charts#238]
Hi,
i upgraded my existing sentry installation using **`helm upgrade`** command with default settings everything else worked fine but the sentry web pod is unable to launch a new pod in parallel as its failed to attach a volume that's already being used by old pod 
Here are the logs i am getting from 'kubectl get events'
`<invalid>   Warning   FailedMount              pod/sentry-web-775f8697c7-4szm8                     Unable to attach or mount volumes: unmounted volumes=[sentry-data], unattached volumes=[default-token-4c5wk config sentry-data]: timed out waiting for the condition`
Deleting and creating pods give us more info. Same error occurs when the web pod is scaled
```
          Warning   FailedAttachVolume   pod/sentry-web-775f8697c7-pvlpv                     Multi-Attach error for volume "pvc-d569e6d4-2b5d-4296-964a-639e012ebe4e" Volume is already used by pod(s) sentry-web-5dccc74756-p6cd2
```
I am using dynamic volume provisioning in my kubernetes cluster 
How can i fix this and autoscale my cluster pods based on load ?

**Comment by Mokto** [comment]
Hi,
This question has already been answered multiple times but, you can :

- not use a PVC
- use a PVC with ReadyWriteMany

**Comment by Hassnain-Alvi** [comment]
How can we use existing PVC for sentry ?
I didn't see any option in https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml

**Comment by Mokto** [comment]
Once again, please look at existing issues.

https://github.com/sentry-kubernetes/charts/issues/63

**Comment by paulchoi** [comment]
I ran into this issue as well.
Based on https://github.com/sentry-kubernetes/charts/issues/238#issuecomment-725433048, and what I've seen, I can assume that:
- only one sentry-web Pod may exist at a given time

So what I ended up doing is:
```
sentry:
  web:
    strategyType: Recreate
```

That way, the previous sentry-web Pod terminates, freeing up the PVC. It's not optimal since that will obviously lead to the web component being offline for a short time.
